server/graphql/batch_mutations/src/batch_prescription.rs

```diff
@@ -17,23 +17,23 @@ use crate::{to_standard_error, VecOrNone};
 ))]
 #[graphql(concrete(
     name = "UpdatePrescriptionResponseWithId",
-    params(prescription::UpdateResponse)
+    params(prescription::update::UpdateResponse)
```

Comment:
to bring these into line with the current convention.

Reply:
thanks!